src/mame/drivers/tsconf.cpp

```diff
@@ -122,7 +122,7 @@ void tsconf_state::tsconf_io(address_map &map)
 	map(0x0077, 0x0077).mirror(0xff00).rw(FUNC(tsconf_state::tsconf_port_77_zctr_r), FUNC(tsconf_state::tsconf_port_77_zctr_w)); // spi data
 	map(0x005f, 0x005f).mirror(0xff00).rw(m_beta, FUNC(beta_disk_device::sector_r), FUNC(beta_disk_device::sector_w));
 	map(0x007f, 0x007f).mirror(0xff00).rw(m_beta, FUNC(beta_disk_device::data_r), FUNC(beta_disk_device::data_w));
-	map(0x00fe, 0x00fe).select(0xff00).rw(FUNC(tsconf_state::spectrum_port_fe_r), FUNC(tsconf_state::tsconf_port_fe_w));
+	map(0x00fe, 0x00fe).select(0xff00).rw(FUNC(tsconf_state::spectrum_ula_r), FUNC(tsconf_state::tsconf_port_fe_w));
```

Comment:
Write function as well

Reply:
It's just rename, no change to functions. The proper decoding is any even port (a0=0) accesses ula, not specifically port 0xfe, so rename makes more sense.

Reply:
Right. But there is FUNC(tsconf_state::tsconf_port_fe_w)) in same line. Is it compilable without renaming this as well?

Reply:
That's your function in tsconf class, so up to you if you want to rename them or not. It's just the functions back in base class (spectrum.cpp) that you're inheriting from that are renamed.